TMDBApp/TMDBApp/ViewModels/MovieViewModel.swift

```diff
@@ -59,4 +60,17 @@ class MovieViewModel: ObservableObject {
             }
         }
     }
+    
+    func loadMovieDetails(movieId: Int) {
+        TMDBService.shared.fetchMovieDetails(movieId: movieId) { [weak self] result in
+            DispatchQueue.main.async {
```

Comment:
Ovo se u modernom Swiftu izbjegava, pogledaj ovaj clanak: https://www.hackingwithswift.com/quick-start/concurrency/how-to-use-mainactor-to-run-code-on-the-main-queue